Cargo.toml

```diff
@@ -21,7 +21,7 @@ gdk = { git = "https://github.com/gtk-rs/gdk" }
 gtk = { git = "https://github.com/gtk-rs/gtk" }
 
 [features]
-default = ["gtk/v3_10"]
+default = ["gtk/v3_16"]
```

Comment:
Why do you suddenly need `v3_16` here?

Reply:
I just replaced. But I guess it's pretty useless in fact... I'll remove it.

Reply:
You should only add such things if you're actually requiring new API from 3.16 :)

Reply:
Good to go after that change